SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3011,10 +3011,23 @@ void USpatialNetDriver::TryFinishStartup()
 				ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(GetWorld()->GetWorldSettings());
 				if (WorldSettings && WorldSettings->bEnableDebugInterface)
 				{
+					auto DebugCompFilter = [this](const Worker_EntityId EntityId, const SpatialGDK::EntityViewElement& Element) {
+						if (Element.Components.ContainsByPredicate(
+								SpatialGDK::ComponentIdEquality{ SpatialConstants::GDK_DEBUG_COMPONENT_ID }))
+						{
+							return true;
+						}
+
+						return false;
+					};
+					TArray<FDispatcherRefreshCallback> DebugCompRefresh = {
+						Connection->GetCoordinator().CreateComponentExistenceRefreshCallback(SpatialConstants::GDK_DEBUG_COMPONENT_ID)
+					};
+
 					// Create the subview here rather than with the others as we only know if we need it or not at
 					// this point.
 					const SpatialGDK::FSubView& DebugActorSubView = Connection->GetCoordinator().CreateSubView(
-						SpatialConstants::GDK_DEBUG_COMPONENT_ID, ActorFilter, ActorRefreshCallbacks);
```

Comment:
removing the actor filter and callbacks could lead to tombstoned actors being passed to the debug system (as well as other reasons you might see the debug tag but the actor itself could be incomplete- asset not loaded yet?) but maybe this is fine. Is this something you already considered when modifying this?

Reply:
It would not matter too much for the debug system itself, but maybe we should keep the filters in place anyway. This is one case where I'm tempted to release it as is, and wait for a bug before taking action.